pkg/handler/repositories.go

```diff
@@ -53,7 +53,7 @@ func RegisterRepositoryRoutes(engine *echo.Group, daoReg *dao.DaoRegistry,
 	addRoute(engine, http.MethodPost, "/repositories/", rh.createRepository, rbac.RbacVerbWrite)
 	addRoute(engine, http.MethodPost, "/repositories/bulk_create/", rh.bulkCreateRepositories, rbac.RbacVerbWrite)
 	addRoute(engine, http.MethodPost, "/repositories/:uuid/introspect/", rh.introspect, rbac.RbacVerbWrite)
-	addRoute(engine, http.MethodGet, "/repositories/:uuid/gpg_key/", rh.getGpgKeyFile, rbac.RbacVerbRead)
+	addRoute(engine, http.MethodGet, "/repository_gpg_key/:uuid", rh.getGpgKeyFile, rbac.RbacVerbRead)
```

Comment:
```suggestion
	addRoute(engine, http.MethodGet, "/repository_gpg_key/:uuid/", rh.getGpgKeyFile, rbac.RbacVerbRead)
```

Reply:
we don't usually put a trailing slash on the endpoints that end with a uuid

Reply:
i mean maybe we should put a trailing slash for everything to be consistent, but we haven't

Reply:
yes, i'm okay without one, but then all the other instances here need to be updated to not have a trailing slash

Reply:
right now the other instances don't have a trailing slash. they look like this, `repositories/:uuid`

Reply:
or really just two places, commented on them :)

Reply:
oh whoops